### ISSUE TITLE
Voiceover: options within dropdowns are not audible

### ISSUE THREAD

**Issue by ghost** [DivanteLtd/next#390]
Turn on voiceover, on a category page tab to the sort by dropdown, observe that you cannot hear the other options in the dropdown by going up and down in the list

**Comment by Qrzy** [comment]
Reported to SFUI team: https://github.com/DivanteLtd/storefront-ui/issues/1068

**Comment by pspaczek** [comment]
related: https://github.com/DivanteLtd/storefront-ui/pull/1095
